proto/api/v1/config_service.proto

```diff
@@ -14,6 +14,46 @@ message PutConfigRequest {
     storage.Config config      = 1;
 }
 
+message DayOption {
+    uint32 num_days = 1;
+    bool   enabled  = 2;
+}
+
+message VulnerabilityDeferralConfig {
+    message FixableCVEOptions {
+        // This options allows users to expire the vulnerability deferral request if and only if **all** vulnerabilities
+        // in the requests become fixable.
+        bool all_fixable = 1;
+        // This options allows users to expire the vulnerability deferral request if **any** vulnerability
+        // in the requests become fixable.
+        bool any_fixable = 2;
+    }
```

Comment:
Is the UX team aware of the distinction here and creating updated mocks? Or is this forward-looking and we will use a static default in the UI for GA?

Reply:
Yes, it was finalized. However, mocks are yet to be available. The UI can be smart about presenting a simpler text when the modal is opened for just one CVE because all and any mean has the same effect in that case.